### DISCUSSION TITLE
LDAP integration doesn't map emails

### DISCUSSION THREAD

**Discussion by AirTobe91** [keycloak/keycloak#15748] [Q&A]
### Describe the bug

Hello,
I have a running keycloak docker container (version 18.0.2).
There I added a LDAP user federation and was able to successfully import all our users and groups from our LDAP (IPA).
I had to adapt the mapping of the first name because it showed the complete name of the user instead just the first name.
The only thing what will not work is the mapping for the e-mail of the users.

I try to map the "mail" attribute from LDAP to "email" attribute from keycloak.
In the LDAP User Object Classes configuration I added "inetOrgPerson,organizationalPerson".

Is there some reason why this is not working?

Thank you!

### Version

18.0.2

**Comment by darius-m** [comment]
Can you maybe add a screenshot of the mapper's configuration?
